clang/lib/Frontend/InitPreprocessor.cpp

```diff
@@ -96,7 +96,7 @@ static void AddImplicitIncludePCH(MacroBuilder &Builder, Preprocessor &PP,
 template <typename T>
 static T PickFP(const llvm::fltSemantics *Sem, T IEEEHalfVal, T IEEESingleVal,
                 T IEEEDoubleVal, T X87DoubleExtendedVal, T PPCDoubleDoubleVal,
-                T IEEEQuadVal) {
+                T BFloatVal, T IEEEQuadVal) {
```

Comment:
I think it makes sense for bfloat to go next to half so that the order continues to go from smallest to largest.